internal/verifier/verifier.go

```diff
@@ -127,7 +127,8 @@ func chdir(dir string) {
 // AmberProvenanceMetadataVerifier verifies Amber provenances by comparing the
 // content of the provenance predicate against a given set of expected values.
 type AmberProvenanceMetadataVerifier struct {
-	provenanceFilePath string
+	Got  *amber.ValidatedProvenance
```

Comment:
Please use `slsa.ValidatedProvenance` here.

Reply:
SG, but `AmberProvenanceMetadataVerifier` might a bit of a confusing name then. Should I try to add a comment/issue explaining?

Reply:
I agree. I think `ProvenanceMetadataVerifier` is a better name. We will no longer have an Amber provenance.